src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,11 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
+    }
+    
+    public static AppUrl getAppUrl(String relativeUrl) {
```

Comment:
The relativeUrl needs a leading slash? Document it as a header comment?
What happens if someone passes a full url instead of a relative one? Ideally, an assertion should fire in that case.